packages/fetch/src/request.js

```diff
@@ -119,6 +119,10 @@ export default class Request extends Body {
 		if (signal != null && !isAbortSignal(signal)) {
 			throw new TypeError('Expected signal to be an instanceof AbortSignal or EventTarget');
 		}
+		
+		if (!signal) {
+			signal = new AbortController().signal;
+		}
```

Comment:
Nit, but maybe we just change line 116 from `null` to `new AbortController().signal` to co-locate the default value with the assignment?

Reply:
We still need to be able to throw the exception in an invalid signal is provided. I.e an empty string is provided should fail and there are tests around this that break if we inline the creation above.

Reply:
interesting - I don't know what path would cause it to fail, since the exception is guarded by `signal != null` so that should only apply to cases that don't reach the `null` but yeah let's keep it separate to be safe 👍